includes/wikia/Extensions.php

```diff
@@ -428,6 +428,10 @@
 	$wgScribuntoUseGeSHi = $wgEnableSyntaxHighlightGeSHiExt;
 	$wgWysiwygDisabledNamespaces[] = NS_MODULE;
 
+	if ( $wgServer === 'runescape' ) {
```

Comment:
`wgServer` is the domain, so for Runescape it would be `http://runescape.fandom.com` and due to language paths, `wgServer` is the same for all language versions. So I suggest changing this to:
```suggestion
	// Change limit on Runescape temporarily (PLATFORM-4090)
	if ( $wgCityId == 304 ) {
```

Reply:
Yeah. Obviously. I must have been drunk when I did this change :(